src/apis/entries_api.rs

```diff
@@ -52,18 +51,29 @@ pub enum SearchLogQueryError {
     UnknownValue(serde_json::Value),
 }
 
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct LogEntries {
+    entries: Vec<LogEntry>,
+}
 
-/// Creates an entry in the transparency log for a detached signature, public key, and content. Items can be included in the request or fetched by the server when URLs are specified. 
-pub async fn create_log_entry(configuration: &configuration::Configuration, proposed_entry: crate::models::ProposedEntry) -> Result<::std::collections::HashMap<String, serde_json::Value>, Error<CreateLogEntryError>> {
+/// Creates an entry in the transparency log for a detached signature, public key, and content. Items can be included in the request or fetched by the server when URLs are specified.
+// Change the return value of the function to LogEntry from ::std::collections::HashMap<String, serde_json::Value>
+pub async fn create_log_entry(
+    configuration: &configuration::Configuration,
+    proposed_entry: crate::models::ProposedEntry,
+) -> Result<LogEntry, Error<CreateLogEntryError>> {
     let local_var_configuration = configuration;
```

Comment:
Why do you need to do that? Both variables are pointing to the same area of memory 🤔

Reply:
I got rid of it now.
Thanks for pointing it out :100: